requirements-pt.txt

```diff
@@ -15,3 +15,4 @@ Pillow>=8.3.2
 tqdm>=4.30.0
 rapidfuzz>=1.6.0
 huggingface-hub>=0.4.0
+fasttext>=0.9.2
```

Comment:
would you mind also adding a relevant upper bound in the specifier please? :pray:

Reply:
@frgfm @aminemindee we should check If we can use `langdetect` package instead of fasttext seems to be more lightweight 👍

Reply:
I first used langdetect but for the same small text the model will predict different languages and using `DetectorFactory.seed = 0` did not solve that issue so i went for fasttext instead. But yeah it's slower. 
For this part about language detection, we can discuss on the new pr with just that part #1023

Reply:
remove :)

Reply:
ah my bad ^^ removed it